Main.java

```diff
@@ -1,80 +1,122 @@
+                    checkWin(player1);
+                }
+                if (playerWinner == true)
+                    break;
+                else {
+                    blockPlayer(player1, chosenPlayer);
+                    printTable();
+                    checkWin(chosenPlayer);
+                }
+            }
+        }
+        if (number == 2) {
+            System.out.println("Give a name for Player 2");
+            name = scanner.next();
+            chosenPlayer = new Player(name, 'O');
+            for (int i = 0; i < 5; i++) {
+                if (playerWinner == true)
+                    break;
+                else {
+                    putSignQuestions(player1);     //V jak to uogólnić player[wybór nr gracza?].putQuestions() - metoda(Player player)
+                    checkWin(player1);
+                }
+                if (playerWinner == true)
+                    break;
+                else {
+                    putSignQuestions(chosenPlayer);
+                    checkWin(chosenPlayer);
+                }
+            }
+        }
+        if (playerWinner == true) {
+            System.out.println();
+            System.out.println("\nTHIS IS THE END OF THE GAME. THANK YOU");
+            return; // return void - zwraca pustke I WTEDY WYCHODZI WCZESNIEJ Z METODY
+        } else {
+            putSignQuestions(player1);
+            checkLastMove(player1);
         }
-        putSignQuestions(player1);
-        checkLastMove(player1);
-        System.out.println("/nTHIS IS THE END OF THE GAME. THANK YOU");
+        System.out.println("\nTHIS IS THE END OF THE GAME. THANK YOU");
     }
 
-    static public void printTable(){
+    static public void printTable() {
+        System.out.println("\nTABLE DESIGN:\n---------");
         for (int i = 0; i < 3; i++) {
-            System.out.println(gameTable[i][0] + " " + gameTable[i][1] + " " + gameTable[i][2]);
+            System.out.println("| " + gameTable[i][0] + " " + gameTable[i][1] + " " + gameTable[i][2] + " |");
         }
+        System.out.println("---------");
     }
 
-    static public void enterCoordinates(Player player){  // TRZEBA ZROBIĆ OBSŁUGĘ WYJĄTKÓW - ARRAYINDEXOUTOFBOUND
-        System.out.println(player.getName() + " Give a column number where you want to put a sign");
-        //int
-        columnNumber = scanner.nextInt();
-        System.out.println(player.getName() + " Give a raw number where you want to put a sign");
+    static public void enterCoordinates(Player player) {  // TRZEBA ZROBIĆ OBSŁUGĘ WYJĄTKÓW - ARRAYINDEXOUTOFBOUND
+        System.out.println("\n" + player.getName() + ", Give a raw number where you want to put a sign");
         //int
         rawNumber = scanner.nextInt();
+        System.out.println("\n" + player.getName() + ", Give a column number where you want to put a sign");
+        //int
+        columnNumber = scanner.nextInt();
```

Comment:
just validate the provided number - if out of bounds then as again for the number informing that number is incorrect